### PR TITLE
setup protractor configuration

### PR DESCRIPTION
**Pull request by david1999erazo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "protactor-workshop-2020",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Protactor",
+  "main": "index.js",
+  "scripts": {
+    "clean": "del /s dist",
```
Comment (yesid-lopez) [inline package.json]: use rm instead...

File: package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "protractor"
+  ],
+  "author": "[David Erazo] <[erazo29@hotmail.com]> ([david1999erazo])",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/david1999erazo/protactor-workshop-2020/issues"
+  },
+  "homepage": "https://github.com/david1999erazo/protactor-workshop-2020#readme",
+  "dependencies": {
+    "protractor": "^7.0.0"
```
Comment (yesid-lopez) [inline package.json]: move protractor as dev dependency

**Comment by yesid-lopez** [comment]
Can you add a Screenshot of the test execution? Thx

**Comment by david1999erazo** [comment]
Test

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**